src/components/ButtonNav/styled.ts

```diff
@@ -9,32 +11,38 @@ export const Wrapper = styled.nav`
 	justify-content: space-around;
 	align-content: center;
 
-	@media (max-width: 50em) {
+	@media (max-width: ${MediaQueries.Ipad}) {
 		position: fixed;
 		bottom: 0;
 		z-index: 999;
 	}
 `
 
-export const StyledButton = styled(Button)`
+type Props = {
+	setColor: string
```

Comment:
This should be named e.g. `hoverColor` or simply `color`, because `setColor` implies that it should be a function. :smile: